schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/message/MailMessage.java

```diff
@@ -14,7 +13,7 @@
  * @author krico
  * @since 26/11/14.
  */
-@Model(kind = "Msg", schemaVersionName = Constants.SCHEMA_VERSION_NAME, schemaVersion = 0)
+@Model
```

Comment:
Because we don't define the kind here, old messages should probably be migrated or we won't be able to access them with this `@Model` anymore.  We could also just delete old ones...